scripts/components/Guild.js

```diff
@@ -4,7 +4,7 @@ import classnames from 'classnames'
 
 import Emoji from './Emoji'
 import GuildIcon from './GuildIcon'
-import chevron from '../../assets/chevron-down-solid.svg'
+import chevron from 'url:../../assets/chevron-down-solid.svg'
```

Comment:
What caused the rename here?

Reply:
Parcel 2 requires those to be moved to `url:`, else, it crashes